syft/frameworks/torch/mpc/aby3/aby3_helper.py

```diff
@@ -0,0 +1,70 @@
+import torch
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+from syft.frameworks.torch.mpc.przs import gen_alpha_3of3
+
+class ABY3Helper:
+    def bit_inject(rst_binary):
+        assert rst_binary.ring_size == 2
+
+        return_wrapper = rst_binary.is_wrapper
+        if return_wrapper:
+            rst_binary = rst_binary.child
+
+        players = rst_binary.players
+        shares_p1, shares_p2, shares_p3 = rst_binary.retrieve_pointers()
+
+        """
+            x shared in ring size 2
+            x_b = (x_b_1, x_b_2, x_b_3)
+
+            Share the shares in ring size L
+            Resharig the binary shars in arithmetic ones should have no communication cost
+            but because we use an orchestrator there is added the communicatio overhead
```

Comment:
```suggestion
            Resharing the binary shares in arithmetic ones should have no communication cost
            but because we use an orchestrator there is added communication overhead
```